larray/viewer/api.py

```diff
@@ -67,8 +67,9 @@ def edit(obj=None, title='', minvalue=None, maxvalue=None, readonly=False, depth
     """
     Opens a new editor window.
 
-    obj : np.ndarray, LArray, Session, dict or str, optional
+    obj : np.ndarray, LArray, Session, dict, str or int, optional
         Object to visualize. If string, array(s) will be loaded from the file given as argument.
+        If -2, reload the last loaded session if any. 
```

Comment:
Users shouldn't see the -2. Implementing it like this is fine (though a specific object might be better), but in any case, I think users should see a more telling constant, eg
```python
from larray import *
edit(LAST_OPEN_FILE)
```
or
```python
import larray as la
la.edit(la.LAST_OPEN_FILE)
```